service/app/telemetry/dispatcher/IDispatcher.java

```diff
@@ -0,0 +1,14 @@
+package telemetry.dispatcher;
+
+import java.util.List;
+
+/**
+ * 
+ * @author Mahesh Kumar Gangula
+ *
+ */
+
+public interface  IDispatcher {
+
+	public void dispatch(List<String> events) throws Exception;
```

Comment:
Mandatory to have Javadoc for interface methods.

Reply:
Interface is deleted.